public/js/eventsPage.js

```diff
@@ -208,24 +210,55 @@ async function displayHiringPartnersAttending(event){
  *
  * @param events an object which contains information about an event
  */
-async function eventGenerator(event, hiringPartners) {
+async function eventGenerator(event, hiringPartners, applicants) {
     let eventInformation = ''
     let date = new Date(event.date).toDateString()
+    let numberOfAttendees
+    if (event.category_name === 'Assessment') {
+        numberOfAttendees = applicants.length
+        console.log(numberOfAttendees)
```

Comment:
This is actually counting all applicants, not just ones assigned to a given assessment day, is this because it's from story 10 which was unfinished?

Reply:
Discussed with you but yes, didn't realise the query in s10 fetches all applicants and then filters them in the js file instead of fetching applicants for a specific event